openapi-contracts/src/test/java/com/sinch/sdk/domains/verification/models/dto/v1/start/request/internal/VerificationStartSmsOptionsTest.java

```diff
@@ -0,0 +1,103 @@
+          .put("a key", "a value")
+          .put("secondKey", "second value")
+          .build();
+
+  public static VerificationStartSmsOptions startVerificationSmsOptionDtoWithAcceptLanguage =
+      VerificationStartSmsOptions.builder()
+          .setExpiry("01:02:03")
+          .setCodeType(VerificationStartSmsOptions.CodeTypeEnum.ALPHANUMERIC)
+          .setTemplate("My template require to use '{{CODE}}' code")
+          .setAcceptLanguage("es-ES")
+          .build();
+
+  @GivenTextResource(
+      "/domains/verification/v1/start/request/internal/VerificationStartSmsOptionsDto.json")
+  String jsonVerificationStartSmsOptions;
+
+  @GivenTextResource(
+      "/domains/verification/v1/start/request/internal/VerificationStartSmsOptionsWithAdditionalPropertiesDto.json")
+  String jsonVerificationStartSmsOptionsWithAdditionalProperties;
+
+  @Test
+  void serializeStartVerificationSmsOptions() throws JsonProcessingException, JSONException {
+    String serializedString = objectMapper.writeValueAsString(startVerificationSmsOptionDto);
+
+    JSONAssert.assertEquals(jsonVerificationStartSmsOptions, serializedString, true);
+  }
+
+  @Test
+  void deserializeStartVerificationSmsOptions() throws JsonProcessingException {
+    Object deserialized =
+        objectMapper.readValue(jsonVerificationStartSmsOptions, VerificationStartSmsOptions.class);
+
+    TestHelpers.recursiveEquals(deserialized, startVerificationSmsOptionDto);
+  }
+
+  @Test
+  void serializeStartVerificationSmsOptionsWithAdditionalProperties()
+      throws JsonProcessingException, JSONException {
+    String serializedString =
+        objectMapper.writeValueAsString(startVerificationSmsOptionWithAdditionalPropertiesDto);
+
+    JSONAssert.assertEquals(
+        jsonVerificationStartSmsOptionsWithAdditionalProperties, serializedString, true);
+  }
+
+  @Test
+  void deserializeStartVerificationSmsOptionsWithAdditionalProperties()
+      throws JsonProcessingException {
+    Object deserialized =
+        objectMapper.readValue(
+            jsonVerificationStartSmsOptionsWithAdditionalProperties,
+            VerificationStartSmsOptions.class);
+
+    TestHelpers.recursiveEquals(
+        deserialized, startVerificationSmsOptionWithAdditionalPropertiesDto);
+  }
+
+  @Test
+  void serializeStartSmsCallWithAcceptLanguage() throws JsonProcessingException, JSONException {
+    String serializedString =
+        objectMapper.writeValueAsString(startVerificationSmsOptionDtoWithAcceptLanguage);
+    // acceptLanguage do not have to be serialized
+    JSONAssert.assertEquals(jsonVerificationStartSmsOptions, serializedString, true);
+  }
+
+  @Test
+  void deserializeStartSmsCallWithAcceptLanguage() throws JsonProcessingException {
+    Object deserialized =
+        objectMapper.readValue(jsonVerificationStartSmsOptions, VerificationStartSmsOptions.class);
+
+    // we are not expecting acceptLanguage to be unserialized because of not sent to network
```

Comment:
Well, it's sent over the network, but not in the body 😉

Reply:
Yes: if comments are present to  to explain something: it have sense to explain right
Done